### PR TITLE
Enable tvm_op for ci

### PR DESCRIPTION
**Pull request by yzhliu**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: tests/python/unittest/test_tvm_op.py [hunk 1]
```diff
@@ -24,14 +24,13 @@
 
 @with_seed()
 def test_tvm_broadcast_add():
-    if _features.is_enabled("TVM_OP"):
```
Comment (marcoabreu) [inline tests/python/unittest/test_tvm_op.py]: Lets keep that if possible

Comment (yzhliu) [reply tests/python/unittest/test_tvm_op.py]: I will, thanks for reminding.  could you guide me where to pip install package in docker? ci failed with "ImportError: No module named 'decorator'" during compiling

**Comment by yzhliu** [comment]
@gigasquid Could you take a look at the following failure? I have no idea what does it mean.

```
:~/incubator-mxnet/contrib/clojure-package/examples/profiler/test$ lein test
INFO  MXNetJVM: Try loading mxnet-scala from native path.
WARN  MXNetJVM: MXNet Scala native library not found in path. Copying native library from the archive. Consider installing the library somewhere in the path (for Windows: PATH, for Linux: LD_LIBRARY_PATH), or specifying by Java cmd option -Djava.library.path=[lib path].
WARN  MXNetJVM: LD_LIBRARY_PATH=null
WARN  MXNetJVM: java.library.path=/usr/java/packages/lib/amd64:/usr/lib/x86_64-linux-gnu/jni:/lib/x86_64-linux-gnu:/usr/lib/x86_64-linux-gnu:/usr/lib/jni:/lib:/usr/lib

lein test core_test
[18:10:20] src/executor/graph_executor.cc:2025: Subgraph backend MKLDNN is activated.

lein test :only core_test/run-profiler

FAIL in (run-profiler) (core_test.clj:30)
expected: (.exists new-file)
  actual: false

lein test :only core_test/run-profiler

FAIL in (run-profiler) (core_test.clj:31)
expected: (> 10000 (- (System/currentTimeMillis) (.lastModified new-file)))
  actual: (not (> 10000 1567188620414))

Ran 1 tests containing 2 assertions.
2 failures, 0 errors.
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/mxnet-scala
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/libtvm_runtime.so
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/libmkldnn.so.0
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/libiomp5.so
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/libmxnet.so
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547/libmklml_intel.so
INFO  org.apache.mxnet.util.NativeLibraryLoader: Deleting /tmp/mxnet158431027493995547
Tests failed.
```

**Comment by gigasquid** [comment]
@yzhliu Sure - I'll pull and take a look this weekend. The test is trying to see if the profiler is working by checking that it wrote a file. Not sure why it would fail.

**Comment by yzhliu** [comment]
Thanks @gigasquid I saw another PR also fails in this test: https://github.com/apache/incubator-mxnet/pull/16046

**Comment by gigasquid** [comment]
@yzhliu I created a PR to fix the flaky test https://github.com/apache/incubator-mxnet/pull/16058. If you cherry pick it into your branch, it should resolve the issues. Please ping me if you have any other problems.

File: ci/docker/Dockerfile.build.ubuntu_gpu_cu101 [hunk 2]
```diff
@@ -80,3 +80,4 @@ RUN /work/ubuntu_adduser.sh
 COPY runtime_functions.sh /work/
 
 WORKDIR /work/mxnet
+ENV LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/cuda/compat
```
Comment (leezu) [inline ci/docker/Dockerfile.build.ubuntu_gpu_cu101]: @yzhliu what is the purpose and need of this line?

**Pull request merged**